services/bots/src/github-webhook/handlers/platinum_review.ts

```diff
@@ -76,7 +76,7 @@ export class PlatinumReview extends BaseWebhookHandler {
           ? `Code owner approval required before merging`
           : `Code owner approval ok.`,
         target_url:
-          'https://developers.home-assistant.io/docs/integration_quality_scale_index/#platinum-',
+          'https://developers.home-assistant.io/docs/core/integration-quality-scale/#-platinum',
```

Comment:
I can no longer find any mention that this is a blocker anymore, @joostlek please confirm.

Reply:
This is still a blocker, the recent IQS upgrade dient change this

Reply:
And where is that described now?
That is what this link needs to point to. The current link does not describe that at all.

Reply:
By the way, should we also approve the description here `Code owner approval required before merging` to `Code owner approval required before merging for Platinum integrations`? That would also make it clearer without leaving github.

Reply:
This is still unresolved; that detail should be added to the linked page, or the linked page should be removed.

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/2595